src/views/tutorial-view/server.ts

```diff
@@ -137,44 +143,47 @@ const moizeOpts: Options = {
 // limit the expensive call for tutorials that have the same product
 const cachedGetAllCollections = moize(getAllCollections, moizeOpts)
 
-export async function getTutorialPagePaths(
-	product: ProductOption
-): Promise<TutorialPagePaths[]> {
-	const allCollections = await cachedGetAllCollections({
-		product: { slug: product },
-	})
-	// Only build collections where this product is the main 'theme'
+export async function getTutorialPagePaths(): Promise<TutorialPagePaths[]> {
+	const allCollections = await cachedGetAllCollections()
```

Comment:
Nothing to do for this comment, but out of curiosity and for the sake of my own learning about the API, would calling `fetchAllCollectionsByProduct` once for each beta product be faster here? I can't remember if `getAllCollections` is one of the endpoints we noted as being slower. Maybe it also becomes less relevant as we onboard each product anyways. 😊

Reply:
I think it would be preferable to limit network requests, even if `getAllCollections` is a large call. This is where the caching becomes helpful.

Reply:
Makes sense! Thank you for the clarity 😊